src/content/self-hosted/manage/air-gapped.mdx

```diff
@@ -45,13 +45,13 @@ Push the following images to your private registry for the Okteto Chart installa
 
 - **okteto/backend:{variables.chartVersion}**
 - **okteto/frontend:{variables.chartVersion}**
-- **okteto/buildkit:{variables.chartVersion}** and **okteto/buildkit:{variables.chartVersion}-rootless**
+- **okteto/buildkit:{variables.chartVersion}** or **okteto/buildkit:{variables.chartVersion}-rootless**
 - **okteto/registry:{variables.chartVersion}**
-- **okteto/pipeline-runner:{variables.chartVersion}**
+- **okteto/pipeline-runner:{variables.chartVersion}** or **okteto/pipeline-runner:{variables.chartVersion}-rootless**
 - **okteto/daemon:{variables.chartVersion}**
-- **registry.k8s.io/ingress-nginx/controller-chroot:{variables.nginxVersion}**
-- **ghcr.io/stakater/reloader:{variables.reloaderVersion}**
-- **bitnami/redis:{variables.redisVersion}**
+- **okteto/ingress-nginx:{variables.chartVersion}**
```

Comment:
this image is `okteto/ingress-nginx-chroot`. The nginx chart adds the `-chroot` if `chroot` is enabled in a [helper](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/_helpers.tpl#L75), so if we want to have all other chroot functions we need to add the `-chroot` to the image

Reply:
Added, thanks!